src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs

```diff
@@ -30,6 +30,7 @@ namespace System.Text.Json
     /// To be able to format the output with indentation and whitespace OR to skip validation, create an instance of 
     /// <see cref="JsonWriterOptions"/> and pass that in to the writer.
     /// </remarks>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public sealed partial class Utf8JsonWriter : IDisposable
 #if BUILDING_INBOX_LIBRARY
```

Comment:
Unrelated to this change, now that we have Microsoft.Bcl.AsyncInterfaces, can we clean up some of this code?

Reply:
Yes, I had pinged @joperezr about this recently. We can also get rid of the Manual ref file:
https://github.com/dotnet/corefx/blob/de1da50fc5f972348aba2fa40ff14e1cbadb13de/src/System.Text.Json/ref/System.Text.Json.Manual.cs#L10

Reply:
@ahsonkhan do we have an issue tracking the work of removing the manual ref file? if so, you can assign it to me.

Reply:
https://github.com/dotnet/corefx/issues/38349